self_teach_challenge_1.1/script.js

```diff
@@ -3,7 +3,11 @@ document.addEventListener('DOMContentLoaded', () => {
 });
 
 let interval;
-let countdownSeconds = 9;
+let countdownSeconds;
+let minutesInput;
+let secondsInput;
+let clicked = true;
+let disable = true;
```

Comment:
Consider: boolean variables are usually names isSomething or hasSomething, so that its obvious that it's a boolean. I would rename these to `wasClicked` and `isDisabled` for example.